packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -110,6 +117,15 @@ export type SelectProps<T> = {
    * @see [tippy.js props](https://atomiks.github.io/tippyjs/v6/all-props/)
    */
   popoverProps?: Omit<PopoverProps, 'onShow' | 'onHide' | 'disabled'>;
+  /**
+   * Enable multiple selection.
+   * @default false
+   */
+  multi?: boolean;
```

Comment:
Maybe name it as a native select has - `multiple`

Reply:
renamed